backend/btrixcloud/crawls.py

```diff
@@ -713,6 +717,17 @@ async def filter_crawl_queue(self, crawl_id, regex):
 
         return num_removed
 
+    async def get_errors_from_redis(self, crawl_id):
+        """Get crawl errors from Redis and optionally store in mongodb."""
+        try:
+            redis = await self.get_redis(crawl_id)
+            errors = await redis.lrange(f"{crawl_id}:e", 0, -1)
+        except exceptions.ConnectionError:
+            # pylint: disable=raise-missing-from
+            raise HTTPException(status_code=500, detail="redis_connection_error")
```

Comment:
500 is usually unexpected crashes, maybe should do 503 or 400 here? I guess 503 is probably most accurate.